tests/integration/test_fasttext_model_artifact.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import tempfile
+import json
+
+import bentoml
+from tests.bento_service_examples.fasttext_classifier import FasttextClassifier
+from bentoml.yatai.client import YataiClient
+
+import fasttext
+
+
+@pytest.fixture()
+def fasttext_classifier_class():
+    FasttextClassifier._bento_service_bundle_path = None
+    FasttextClassifier._bento_service_bundle_version = None
+    return FasttextClassifier
+
+
+test_json = json.dumps({"text": "foo"})
```

Comment:
Ahhh. I think if you just omit `json.dumps` here, you should be fine :)

Reply:
Thanks for catching that! I've made the changes accordingly!.